### PR TITLE
feat: side navigation

### PR DESCRIPTION
**Pull request by Sardar1208**
**- What I did - added side navigation acc to new UI for desktop, created a new folder structure for new screens**
ticket link - https://github.com/atsign-foundation/atmosphere_pro/issues/294

### CODE REVIEW

File: lib/desktop_screens_new/welcome_screen/widgets/sidebar_item.dart [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    await launchUrl(
+      Uri(
+        scheme: 'mailto',
+        path: email,
+      ),
+    );
+  }
+}
+
+class BuildSidebarIconTitle extends StatelessWidget {
```
Comment (sachins-geekyants) [inline lib/desktop_screens_new/welcome_screen/widgets/sidebar_item.dart:167]: New component should be in a new file.

Comment (Sardar1208) [reply lib/desktop_screens_new/welcome_screen/widgets/sidebar_item.dart:167]: Its a local component that is only used by the main component in that file

File: lib/desktop_screens_new/welcome_screen/desktop_home_screen.dart [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import 'package:at_backupkey_flutter/utils/size_config.dart';
+import 'package:atsign_atmosphere_pro/desktop_screens_new/welcome_screen/widgets/sidebar_new.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../../desktop_routes/desktop_route_names.dart';
```
Comment (sonle-geekyants) [inline lib/desktop_screens_new/welcome_screen/desktop_home_screen.dart]: Please use absolute path

**Review by sonle-geekyants — CHANGES_REQUESTED** [review]

File: lib/desktop_screens_new/welcome_screen/desktop_home_screen.dart (hunk 2 above)
Comment (Sardar1208) [reply lib/desktop_screens_new/welcome_screen/desktop_home_screen.dart]: Fixed

**Review by sachins-geekyants — APPROVED** [review]

**Pull request merged**